features/table.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: Tables
-description: The `<table>` element TODO.
+description: The `<table>` HTML element represents tabular data—that is, information presented in a two-dimensional table comprised of rows and columns of cells containing data.
```

My suggestions here are toward brevity and acknowledging that this is more than just one element. What do you think of this instead?

```suggestion
description: The `<table>` HTML element, with several related elements, represents tabular data in rows and columns of cells.
```